apps/react-18-tests-v9/jest.config.js

```diff
@@ -6,6 +6,12 @@
 module.exports = {
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31716&commitId=d334568048c7c35589791b1ee9b265e75adc8be5&env=prod&clientType=FLUENTUI)


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Outline.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline.selected.chromium.png|0| Added|

</details>


  <details><summary><b>Image Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image Converged.Image Variations Fallback.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Removed|
|SwatchPicker Converged.Spacing.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>